var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -35,7 +35,7 @@ class SuiteSparse(Package):
     version('4.5.3', '8ec57324585df3c6483ad7f556afccbd')
     version('4.5.1', 'f0ea9aad8d2d1ffec66a5b6bfeff5319')
 
-    variant('tbb', default=True, description='Build with Intel TBB')
```

Comment:
Building TBB was not a problem, but it throws 'tbb/task_scheduler_init.h' file not found error. Please find more details below. What  do you think it is the best way to merge?

[eklee@c712f5n13 spack]$ spack install tbb%xl_r

make[1]: Entering directory `/tmp/eklee/spack-stage/spack-stage-rjMhGI/SuiteSparse/SPQR'
( cd Lib ; make install )
make[2]: Entering directory `/tmp/eklee/spack-stage/spack-stage-rjMhGI/SuiteSparse/SPQR/Lib'
/usr/bin/xlc++_r -fPIC -DBLAS_NO_UNDERSCORE   -O3 -fexceptions -fPIC -fopenmp -DHAVE_TBB   -I../../CHOLMOD/Include -I../../SuiteSparse_config -I../Include -c ../Source/spqr_parallel.cpp
../Source/spqr_parallel.cpp:10:10: fatal error: 'tbb/task_scheduler_init.h' file not found
#include <tbb/task_scheduler_init.h>
         ^
1 error generated.
Error while processing ../Source/spqr_parallel.cpp.
make[2]: *** [spqr_parallel.o] Error 1
make[2]: Leaving directory `/tmp/eklee/spack-stage/spack-stage-rjMhGI/SuiteSparse/SPQR/Lib'
make[1]: *** [install] Error 2